nidirect_common/nidirect_common.install

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/**
+ * @file
+ * Install, update and uninstall functions for the NIDirect common module.
+ */
+
+use Drupal\user\Entity\User;
+
+/**
+ * Implements hook_install().
+ *
+ * Perform actions to set up the site for this module.
+ */
+function nidirect_common_install() {
+  // Create some test users for the Nightwatch tests that will come along later.
+  $name_list = ['_news_super' => 'news_supervisor',
+    '_gp_author' => 'gp_author_user',
+    '_gp_super' => 'gp_supervisor_user',
+    '_apps' => 'apps_user',
+    '_hc_author' => 'health_condition_author_user',
+    '_hc_super' => 'health_condition_supervisor_user',
+    '_driving_super' => 'driving_instructor_supervisor_user',
+  ];
+  foreach ($name_list as $name => $role) {
+    // Add prefix from environment var.
+    $prefix = getenv('NW_TEST_USER_PREFIX');
+    $password = getenv('TEST_PASS');
+    // If prefix not set, do not create users.
+    if (empty($prefix) || empty($password)) {
+      \Drupal::logger('nidirect_common')->notice(
```

Comment:
I'd be tempted to use 'error' vs 'notice'

Reply:
Good spot, thanks